### PR TITLE
added try/catch clause to solve product issue

### LINKED ISSUE #15: Bug: Division by zero error on products
# CRITICAL ISSUE

When the client requests a single product, or all products, the status code is 500 and the response is always the following.

```
ZeroDivisionError at /products
division by zero
```

The most recent feature added by the team was adding an average rating key to each product. You might want to start there and see if there are any `try/catch` blocks you can implement.

### PR DESCRIPTION
**Pull request by kritikillz931**
Division by zero error solved in products model

## Changes

- added try/catch clause to products model

## Requests / Responses

If this PR contains code that defines a new request/response, or changes an existing one, please put the JSON representations here.

**Request**

GET `/products/50` Gets product with id of 50

**Response**

HTTP/1.1 201 OK

``
{
    "id": 50,
    "name": "Escalade EXT",
    "price": 926.92,
    "number_sold": 2,
    "description": "2008 Cadillac",
    "quantity": 2,
    "created_date": "2019-02-01",
    "location": "Lokavec",
    "image_path": null,
    "average_rating": 3.25
}
```

## Testing

Description of how to test code...

- [ ] git fetch --all
- [ ] change branch to 15-division-by-zero
- [ ] Use postman for testing
- [ ] login with credentials (username = steve) || (password = Admin8*) to get a token
- [ ] make a GET request to http://localhost:8000/products/50
- [ ] confirm your response is equal to the response above


## Related Issues

- Fixes #15

### CODE REVIEW

File: bangazonapi/views/cart.py [hunk 1]
```diff
@@ -54,11 +54,6 @@ def destroy(self, request, pk=None):
         open_order = Order.objects.get(
             customer=current_user, payment_type=None)
 
-        line_item = OrderProduct.objects.filter(
```
Comment (hannahhall) [inline bangazonapi/views/cart.py:57]: These lines shouldn't be removed

**Review by hannahhall — CHANGES_REQUESTED** [review]
Your fix for the ticket looks good, just add those lines back in

File: bangazonapi/views/cart.py (hunk 1 above)
Comment (kritikillz931) [reply bangazonapi/views/cart.py:57]: added those lines back and pushed

**Review by hannahhall — APPROVED** [review]
👍

**Pull request merged**